internal/pullbox/tar.go

```diff
@@ -0,0 +1,88 @@
+// Copyright 2023 Jetpack Technologies Inc and contributors. All rights reserved.
+// Use of this source code is governed by the license in the LICENSE file.
+
+package pullbox
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"syscall"
+
+	"github.com/pkg/errors"
+)
+
+var ErrFileExists = errors.New("file already exists")
+
+// extract decompresses a tar file and saves it to the specified directory
```

Comment:
is this comment correct? It doesn't save to a specified dir....its a temp dir, no?